public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="overflow: overlay">
```

Comment:
This removes header shifting when a scrollbar appears/disappears on the page when you change page.